pkg/sdk/streams_validations_gen.go

```diff
@@ -29,21 +27,21 @@ func (opts *CreateOnTableStreamOptions) validate() error {
 		errs = append(errs, errOneOf("CreateOnTableStreamOptions", "IfNotExists", "OrReplace"))
 	}
 	if valueSet(opts.On) {
-		if ok := exactlyOneValueSet(opts.On.At, opts.On.Before); !ok {
+		if !exactlyOneValueSet(opts.On.At, opts.On.Before) {
 			errs = append(errs, errExactlyOneOf("CreateOnTableStreamOptions.On", "At", "Before"))
 		}
 		if valueSet(opts.On.Statement) {
```

Comment:
There should be an error when it's not set

Reply:
But Statement is not a pointer here, so when it's not set, it fails on `!exactlyOneValueSet...`